.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule ".scripts"]
+	path = .scripts
+	url = https://github.com/libero/scripts
```

Comment:
name can be changed if there's a better fit

Reply:
Probably fine, not that specific but could cover multiple things (updates, testing, deployment etc). Unlikely to conflict too.